src/Routes.js

```diff
@@ -1,36 +1,33 @@
 const modelPage = require('./json/modelPage.json');
 
-/* eslint-disable max-len */
-const routes = {
+export const routes = {
+  NotFound: {
+    url: '/404/',
+    component: 'src/pages/404.js',
+  },
   HomePage: {
     url: '/',
     component: 'src/pages/index.js',
-    seo: { title: 'Home', description: '' },
   },
   AboutPage: {
-    url: '/about',
+    url: '/about/',
```

Comment:
How does this router handle trailing slashes? Will the non-trailing slash version get redirected to the trailing slash version?

Reply:
Yes it get redirected

I was the way to fix double routes on Google Analytics (they made a difference between `about` and `about/`)